frontend/template-mixins/partials/forms/textarea-group.html

```diff
@@ -18,10 +18,10 @@
         <textarea name="{{id}}" id="{{id}}"
             class="govuk-textarea {{#isMaxlengthOrMaxword}}govuk-js-character-count{{/isMaxlengthOrMaxword}} {{#className}}{{className}}{{/className}} {{#error}}govuk-input--error{{/error}}"
             {{#isMaxlengthOrMaxword}}
-            aria-describedby="{{id}}-info"
+            aria-describedby="{{id}}-hint {{id}}-info"
```

Comment:
do we still need {{id}}-info?

Reply:
Yes id-info is the aria 'live' description of the number of characters remaining